internal/sentry/capture.go

```diff
@@ -0,0 +1,101 @@
+package sentry
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/getsentry/sentry-go"
+	sentryecho "github.com/getsentry/sentry-go/echo"
+	"github.com/labstack/echo/v4"
+)
+
+func CaptureException(ctx context.Context, err error,
+	logger echo.Logger, sentryOn bool, noHubMsg string,
+) {
+	if err == nil {
+		return
+	}
+
+	if !sentryOn {
```

Comment:
Here, the error log is only output when `sentryOn` is false. I want to print the error log regardless of whether sentry is on or off. What do you think?

Reply:
Hmm, I guess users don't necessarily expect the capture funcs to call `echo.Logger.Error()` or `Info()`.
In my projects  `Errorj()` or `Infoj()` is usually used for structured log output.

Reply:
you mean in the project we can do it like this?
```
bean. Errorj(...)
sentry.CaptureException(...)
```
but if you set `sentryOn` off, the error log will print twice in the above case.

Reply:
i.c. It's ok for me.
but can you wrap some function contains both `logger.error` and  `sentry.CaptureException`? 
In my opinion, `sentry.CaptureException` may be lost in some cases, but `logger.error` rarely happens, so I want to record the error in two ways ->  `logger.error` and  `sentry.CaptureException`

Reply:
OK, what about if we provide another wrap func like `captureAndLogException` ?

Reply:
perfect for me

Reply:
Nice, let me create it.